code-challenges/arrays.md

```diff
@@ -156,3 +156,144 @@ same_first_last([1, 2, 1]) → true
 function same_first_last(arr) {
     return arr.length >= 1 && arr[0] == arr[array.length -1];
 }
+
+#### Problem 11 - 
+
+Given an array of ints, return a new array containing only the odd numbers in the same order that they appeared in the original array.
+
+find_odds([1,2,3,4,5,6]) → [1,3,5]
+find_odds([7,1,9,8,5,2,4,6]) → [7,1,9,5]
+find_odds([2,4,6,7]) → [7]
+
+Sample Solution: 
+
+function find_odds(arr) {
+   return arr.filter((num) => num % 2 === 1);
+}
```

Comment:
~~~suggestion
Given an array of ints, return a new array containing only the odd numbers in the same order that they appeared in the original array.

```
find_odds([1,2,3,4,5,6]) → [1,3,5]
find_odds([7,1,9,8,5,2,4,6]) → [7,1,9,5]
find_odds([2,4,6,7]) → [7]
```

Sample Solution: 

```javascript
function find_odds(arr) {
   return arr.filter((num) => num % 2 === 1);
}
```
~~~